azure_arc_data_jumpstart/gke/terraform/artifacts/Bootstrap.ps1

```diff
@@ -74,25 +81,5 @@ Register-ScheduledTask -TaskName "DataServicesLogonScript" -Trigger $Trigger -Us
 # Disabling Windows Server Manager Scheduled Task
 Get-ScheduledTask -TaskName ServerManager | Disable-ScheduledTask
 
-# Changing to Client VM wallpaper
```

Comment:
Hey @dkirby-ms, just wanted to keep it consistent with the other scenarios.

Personally, when I saw the wallpaper "change" when going through the scenario the first time, that was a mental queue that everything is "done" - so that's why I thought it might make sense to move to the end of DataServices (so that if something didn't work, the wallpaper wouldn't show jumpstart).

Hope that makes sense!.